plugin/pkg/scheduler/scheduler.go

```diff
@@ -241,6 +242,39 @@ func (sched *Scheduler) assume(assumed *v1.Pod, host string) error {
 	// If the binding fails, scheduler will release resources allocated to assumed pod
 	// immediately.
 	assumed.Spec.NodeName = host
+
+	if utilfeature.DefaultFeatureGate.Enabled(features.VolumeTopologyBinding) {
```

Comment:
Could you please place this code in a separate function and call it here?

Reply:
Done